vars/goRelease.groovy

```diff
@@ -31,15 +31,11 @@ def call(body) {
     dir(buildPath) {
         git "https://github.com/${ghOrg}/${prj}.git"
 
-        sh "git config user.email fabric8-admin@googlegroups.com"
-        sh "git config user.name fabric8-release"
+        flow.setupGitSSH()
         sh "git remote set-url origin git@github.com:${ghOrg}/${prj}.git"
 
         container(name: 'go') {
             stage ('build binary'){
-                sh 'chmod 600 /root/.ssh-git/ssh-key'
```

Comment:
dropping these lines totally?

Reply:
Though https://github.com/fabric8io/fabric8-pipeline-library/pull/396/files#diff-0008f0d28f3aed4072c34ae9bbda634aR34 going to make sure

Reply:
@hrishin @rupalibehera I'm not very sure about this one. Should the flow.setupGitSSH() command be inside or outside the go container?